frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -259,12 +332,13 @@ function ReaderBookmark:onShowBookmark()
             end
         end
         if v.text == nil or v.text == "" then
+            self.garbage = k
```

Comment:
This garbage is not used :)

Reply:
That's right. I only use it to keep the code inspection of my IDE silent, complaining otherwise about an unused variable k. But I'll remove it.